README.md

```diff
@@ -39,24 +41,27 @@ First, you need to mount the gem in the routes file like this
 # config/routes.rb
 
 Rails.application.routes.draw do
-  mount_graphql_devise_for 'User', at: 'api/v1', authenticable_type: Types::CustomUserType, operations: {
-    login: Mutations::Login
-  }, skip: [:sign_up]
+  mount_graphql_devise_for 'User',
+    at: 'api/v1',
+    authenticable_type: Types::MyCustomUserType,
+    operations: {
+      login: Mutations::Login
+    },
+    skip: [:sign_up]
 end
 ```
 If you used DTA's installer you will have to remove the `mount_devise_token_auth_for`
 line.
 
-Here are the option for the mount method:
+Here are the options for the mount method:
 
-1. `at`: Route where the GraphQL schema will be mounted on the Rails server. In the
-example your API will have this two routes `POST /api/v1/graphql_auth` and `GET /api/v1/graphql_auth`.
-If no this option is not specified, the schema will be mounted at `/graphql_auth`.
-1. `operations`: Specifying this one is optional. Here you can override default
+1. `at`: Route where the GraphQL schema will be mounted on the Rails server. In this example your API will have these two routes: `POST /api/v1/graphql_auth` and `GET /api/v1/graphql_auth`.
+If this option is not specified, the schema will be mounted at `/graphql_auth`.
+1. `operations`: Specifying this is optional. Here you can override default
 behavior by specifying your own mutations and queries for every GraphQL operation.
-Check available operations in this file [mutations](https://github.com/graphql-devise/graphql_devise/blob/b5985036e01ea064e43e457b4f0c8516f172471c/lib/graphql_devise/rails/routes.rb#L19)
-and [queries](https://github.com/graphql-devise/graphql_devise/blob/b5985036e01ea064e43e457b4f0c8516f172471c/lib/graphql_devise/rails/routes.rb#L41).
-All mutations and queries are build so you can extend default behavior just by extending
+Check available operations in the [mutation](https://github.com/graphql-devise/graphql_devise/blob/b5985036e01ea064e43e457b4f0c8516f172471c/lib/graphql_devise/rails/routes.rb#L19)
```

Comment:
This is not actually "the" mutation. It means go to this file to see mutations and queries, each with a link tot he correct line. Original text is correct

Reply:
Got it. I thought you meant to say "these files _mutation_ and _query_" where they were links. I'll revert that.